src/figuresClasses.ts

```diff
@@ -2,10 +1,109 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color;
+
+  constructor(
+    color: string,
+
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
+
+    if (a <= 0) {
+      throw new Error('`a` can`t be lower or equal zero');
+    }
 
-export class Triangle implements Figure {}
+    if (b <= 0) {
+      throw new Error('`b` can`t be lower or equal zero');
+    }
+
+    if (c <= 0) {
+      throw new Error('`c` can`t be lower or equal zero');
+    }
+
+    const longest: number = Math.max(a, b, c);
```

Comment:
The variables `a`, `b`, and `c` are not defined within this scope. You should use `this.a`, `this.b`, and `this.c` to refer to the class properties.